code/modules/library/bibles.dm

```diff
@@ -0,0 +1,352 @@
+GLOBAL_LIST_INIT(biblenames, list(
+	"Bible",
+	"Quran",
+	"Scrapbook",
+	"Burning Bible",
+	"Clown Bible",
+	"Banana Bible",
+	"Creeper Bible",
+	"White Bible",
+	"Holy Light",
+	"The God Delusion",
+	"Tome",
+	"The King in Yellow",
+	"Ithaqua",
+	"Scientology",
+	"Melted Bible",
+	"Necronomicon",
+	"Insulationism",
+	"Guru Granth Sahib",
+	"Kojiki",
+))
+//If you get these two lists not matching in size, there will be runtimes and I will hurt you in ways you couldn't even begin to imagine
+// if your bible has no custom itemstate, use one of the existing ones
+GLOBAL_LIST_INIT(biblestates, list(
+	"bible",
+	"koran",
+	"scrapbook",
+	"burning",
+	"honk1",
+	"honk2",
+	"creeper",
+	"white",
+	"holylight",
+	"atheist",
+	"tome",
+	"kingyellow",
+	"ithaqua",
+	"scientology",
+	"melted",
+	"necronomicon",
+	"insuls",
+	"gurugranthsahib",
+	"kojiki",
+))
+GLOBAL_LIST_INIT(bibleitemstates, list(
+	"bible",
+	"koran",
+	"scrapbook",
+	"burning",
+	"honk1",
+	"honk2",
+	"creeper",
+	"white",
+	"holylight",
+	"atheist",
+	"tome",
+	"kingyellow",
+	"ithaqua",
+	"scientology",
+	"melted",
+	"necronomicon",
+	"kingyellow",
+	"gurugranthsahib",
+	"kojiki",
+))
+
+/obj/item/book/bible
```

Comment:
these dont seem to have storage created anywhere

Reply:
oops my bad, forgot to give them carve_out() on init after removing the hollow books